tests/zep/main.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for Xbee S1 network device driver
```

Comment:
Now I see you introduced this test not long ago. 1. the "old" shell_init produces a warning. 2. how can I use this test? I mean, pinging and so on...